BuildScripts/mvApplicationFunctions.sql

```diff
@@ -152,6 +167,17 @@ DECLARE
 BEGIN
 
     rConst      := mv$buildAllConstants();
+	
+	IF pParallel = 'Y' THEN
+	
+		IF pParallelColumn IS NULL AND pParallelColumn IS NULL AND pParallelJobs = 0 THEN
```

Comment:
IF pParallelColumn IS NULL OR **pParallelAlias** IS NULL OR pParallelJobs = 0 THEN

Reply:
good spot - changed.